experiments/snippets/autoLoadMore/autoLoadMore.xml

```diff
@@ -0,0 +1,86 @@
+<skuidpage unsavedchangeswarning="yes" personalizationmode="server" showsidebar="false" showheader="false">
+	<models>
+		<model id="Controller" limit="20" query="true" createrowifnonefound="true" datasource="Ui-Only" processonclient="true">
+			<fields>
+				<field id="showLoadingMore" displaytype="BOOLEAN" ogdisplaytype="TEXT" defaultvaluetype="fieldvalue"/>
+			</fields>
+			<conditions/>
+			<actions/>
+		</model>
+		<model id="Accounts" query="true" createrowifnonefound="false" datasource="salesforce" limit="15" sobject="Account">
+			<fields>
+				<field id="RecordTypeId"/>
```

Comment:
Let's remove `RecordTypeId` so this page is more portable. It threw an error in my org since I hadn't turned on record types. Since we don't use it for this demo, I think we're save without it.

You can just remove this line from the XML.

Reply:
no problem, removed.  Skuid auto adds it though, always wondered why...